collect_app/src/main/res/drawable/ic_baseline_refresh_error_24.xml

```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
This is just showing the same refresh button in the error color right now. I think ultimately we want a "badge" but if everyone can live with this initially it'll save time faffing around with the new beta badging API (or in creating a new icon).